flatpak/com.mitchellh.ghostty.yml

```diff
@@ -0,0 +1,114 @@
+app-id: com.mitchellh.ghostty
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ziglang
+default-branch: tip
+command: ghostty
+finish-args:
+  # 3D rendering
+  - --device=dri
+  # Windowing
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  # Allow user to specify additional config files in home by default
+  - --filesystem=home:ro
+  # So we can escape the sandbox
+  - --talk-name=org.freedesktop.Flatpak
+cleanup:
+  - /include
+  - /lib/girepository-1.0
+  - /lib/pkgconfig
+  - /share/gir-1.0
+  - /share/pkgconfig
+  - /share/vala
+  - '*.la'
+  - '*.a'
+  - '*.so'
+
+modules:
+  - name: bzip2-redirect
+    buildsystem: simple
+    build-commands:
+      - install -dm755 /app/lib
+      - echo "INPUT(libbz2.so)" > /app/lib/libbzip2.so
+
+  - name: blueprint-compiler
+    buildsystem: meson
+    cleanup:
+      - '*'
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/jwestman/blueprint-compiler.git
+        tag: v0.16.0
+        commit: 04ef0944db56ab01307a29aaa7303df6067cb3c0
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
+
+  - name: gtk4-layer-shell
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/wmww/gtk4-layer-shell.git
+        tag: v1.1.0
+        commit: 93550245220cdc514be4701b517acd374a86acc2
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([\d.]+)$
```

Comment:
@pluiedev @jcollie how can we best make sure that these are matching the build.zig.zon or where ever else we track these? A CI job? Generating the manifest? CI job sounds better than generating, but that's just me opinion.

Reply:
Can we use plain `https` downloads rather than git checkouts? If so it would be fairly easy to keep those in sync when we update the Nix deps. See `build.zig.zon.txt`

Reply:
hmm yeah, i think that the plan is to make blueprint compiler optional for packagers by precompiling all of the blueprints to .ui files. then unless you're really hardcore you wouldn't need blueprint compiler at all

Reply:
gtk4-layer-shell is now using archive.

I'd prefer that we get the installation bug fixed so I can just use the bundled version, but I haven't figured out how to make it work.